kcsd/tests/test_oKCSD.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# encoding: utf-8
+from __future__ import print_function, division, absolute_import
+import os
+import unittest
+import numpy as np
+
+from kcsd import oKCSD2D, oKCSD3D
+
+try:
+    basestring
+except NameError:
+    basestring = str
+
+
+n_src = 5
+
+class testsKCD(unittest.TestCase):
+    def setUpClass(self):
```

Comment:
setUpClass as a method name is confusing. Please divide this function into two separate functions testing initialization of oKCSD2D and oKCSD3D and name them accordingly (test_oKCSD2D_initialization and test_oKCSD3D_initialization).